targets/target_persistent.c

```diff
@@ -14,6 +21,39 @@ static int enumerate_ranges(const GumRangeDetails* details, gpointer user_data)
     return 0;
 }
 
+static void prefetch_log(const GumEvent* event, GumCpuContext* cpu_context, gpointer user_data) {
+    switch (event->type) {
+    case GUM_COMPILE: {
+        const GumCompileEvent* compile = &event->compile;
```

Comment:
You could consider doing this in the transformer to avoid the need for a separate event sink. This may help performance.